app/src/main/res/values/styles.xml

```diff
@@ -6,13 +6,18 @@
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
+        <item name="autoCompleteTextViewStyle">@style/cursorColor</item>
```

Comment:
You don't need to change cursor color or style, the one I requested was about hint text

Reply:
at first I didn't change it, but when I click on the button, the cursor is not visible. I think it doesn’t look comfortable.

Reply:
Same as above, should be achievable with styles/themes, check that option.